tests/test_reader.py

```diff
@@ -61,10 +61,12 @@ def test_read_cobayamcmc():
         np.loadtxt("./tests/example_data/cb.2.txt", usecols=0)
     ))
     assert_array_equal(mcmc.weights, w)
-    params = rp_cb("./tests/example_data/cb")
+    params, tex = rp_cb("./tests/example_data/cb")
     params.append('logL')
     params.append('chain')
     assert_array_equal(mcmc.columns, params)
+    tex['chain'] = r'$n_\mathrm{chain}$'
+    assert mcmc.tex == tex
```

Comment:
It might be better to have an explicit list of tex labels to test both `rp_cb` and `mcmc` against (and probably similarly for `read_getdist`), since this did not catch the failed reading in the current setup.

Reply:
Agreed. We no longer call the readers, and test explicitly against the tex. Tying the tex labels to a column in #215 should hopefully make this much safer.